Uplift/Controllers/FilterViewController.swift

```diff
@@ -160,6 +160,7 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
 
         classTypeDropdownData = DropdownData(completed: false, dropStatus: .closed, titles: [])
 
+        /*
```

Comment:
Is there a reason why you commented using the /* ... */ here? Just wondering because you were using // in previous comments